tests/expectations.json

```diff
@@ -11019,18 +11003,7 @@
   "built-ins/TypedArray/prototype/with/this-value-invalid.js": "CRASH",
   "built-ins/TypedArray/resizable-buffer-length-tracking-1.js": "CRASH",
   "built-ins/TypedArray/resizable-buffer-length-tracking-2.js": "CRASH",
-  "built-ins/TypedArrayConstructors/BigInt64Array/prototype/BYTES_PER_ELEMENT.js": "CRASH",
   "built-ins/TypedArrayConstructors/BigUint64Array/prototype.js": "CRASH",
```

Comment:
**Unrelated:** Why does the `BigUint64Array` prototype test still fail yet all other TypedArray prototype tests don't?

Reply:
Hmm. That's a good question.

Reply:
Alright: Some failures at least are caused by the error that is thrown if you run `String(new BigUint64Array())`.

Effectively, it's expected because of our `print` function's method.